### PR TITLE
pins all 4.5 dependencies and runs hack/update-deps.sh

### PR DESCRIPTION
**Pull request by p0lyn0mial**
This PR proves that `origin-4.5-kubernetes-1.18.3` branches used by this branch (`release-4.5`) are in a good condition and in sync with `vendor/k8s.io`

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -109,13 +109,13 @@ import:
 - package: github.com/ugorji/go
   version: d75b2dcb6bc8
 
-# openshift second
+# openshift second, pinned to specific revisions to limit bringing in new code when upgrading dependencies
```
Comment (p0lyn0mial) [inline glide.yaml:112]: added manually

File: glide.yaml [hunk 2]
```diff
@@ -204,3 +204,379 @@ import:
   version: v1.0.1
 - package: github.com/andreyvit/diff
   version: c7f18ee00883
+
+  # generated by tools/depcheck
```
Comment (p0lyn0mial) [inline glide.yaml:208]: added by `depcheck` tool

**Comment by eparis** [comment]
I'm manually adding valid-bug.  Will need to hold until 4.5.1 is open for cherry-pick approval from sdodson.

**Comment by sdodson** [comment]
/hold
To stop the spinning on a test job.

**Comment by p0lyn0mial** [comment]
@sdodson since all jobs passed, could you add the `cherry-pick-approved` label back and unhold the PR?

**Pull request merged**